src/nodeFinder.cpp

```diff
@@ -575,6 +579,90 @@ NAN_METHOD(Finder::IsWorkplan)
     return;
 }
 
+NAN_METHOD(Finder::IsWorkplanWIthSetupAndFixture)
+{
+    Finder * find = Nan::ObjectWrap::Unwrap<Finder>(info.This());
+    if (find == 0) //Throw Exception
+	return;
+
+    if (info[0]->IsUndefined())
```

Comment:
Make sure when you're checking if arguments are valid, to use `info.Length() > x` rather than using `info[0].IsUndefined()`